src/binaryen-c.cpp

```diff
@@ -4968,7 +4968,6 @@ BinaryenTableRef BinaryenAddTable(BinaryenModuleRef module,
                                   BinaryenIndex maximum,
                                   BinaryenType tableType) {
   auto table = Builder::makeTable(name, Type(tableType), initial, maximum);
-  table->hasExplicitName = true;
```

Comment:
Why change this?

Reply:
I thought it is no longer needed since `Builder::makeTable` now uses explicit name by default.